.github/workflows/ci.yml

```
@@ -111,6 +111,8 @@ jobs:
         uses: ScribeMD/docker-cache@0.5.0
         with:
           key: ddev-images-${{ runner.os }}-${{ steps.ddev_version.outputs.ddev_version }}-${{ hashFiles('.ddev/config.yaml', '.ddev/docker-compose.cypress.yaml', '.ddev/docker-compose.redis.yaml') }}
+          read-only: ${{ github.event_name == 'pull_request' }}
```

Another thought: this condition makes this cache read-only for all PRs. This means it is not read-only for every other event type. This also means that if someone pushes a tag, it will create a new cache entry. I know this is a narrow case but when we're doing this, why not address that issue as well?

One way I see around this is to change the condition to check for the ref name to not match `main`. Something like:

```suggestion
          read-only: ${{ github.ref_name != 'main' }}
```

---

Yes makes sense. I considered using `github.event_name != 'push'` so the action would work for any branch, since branches could be named `master`, `main`, `developer`, or `stage`. However, tag events are also triggered by `push`, so let's go with `github.ref_name != 'main'`. We can add a comment, If branch names are different, adjust the condition accordingly. For multiple branches, use a condition like `github.ref_name != 'main' && github.ref_name != 'develop'`.

---

Good catch. I now think that the tags scenario is less likely than the branch named something else. Should we just keep the existing condition?

---

I found another way so there is `github.ref_type` which is 1branch1 for other operation and 1tag1 for tags push so we can change our condition, `github.event_name == 'pull_request' || github.ref_type == 'tag'`